### ISSUE TITLE
preprocessing of VCC 2016 dataset

### ISSUE THREAD

**Issue by frankdarkluo** [sky1456723/Pytorch-MBNet#1]
Is it okay to provide the csv file of VCC 2016 dataset for calculating the VCC 2016 system-level LCC and SRCC and MSE?

**Comment by sky1456723** [comment]
Yes, I can provide the csv file of VCC 2016 and the testing code for VCC 2016 after cleaning the code. 
In the original design, testing on VCC 2016 can be done by using inference.py (to get the inference results) and a simple code (to sum up the results for each system, customized by the users).
